src/sdk/pynni/nni/multi_phase/multi_phase_tuner.py

```diff
@@ -60,6 +60,13 @@ def receive_customized_trial_result(self, parameter_id, parameters, value, trial
         """
         _logger.info('Customized trial job %s ignored by tuner', parameter_id)
 
+    def trial_end(self, parameter_id, success, trial_job_id):
+        """Invoked when a trial is completed or terminated. Do nothing by default.
+        parameter_id: int
```

Comment:
update the docstring about "trial_job_id"?

Reply:
will fix. thanks~